pkg/manager/config/consts.go

```diff
@@ -0,0 +1,28 @@
+package config
+
+import "time"
+
+const (
+	// LeaderElectionEnabled is a constant that represents a value that should be used to enable leader election.
+	LeaderElectionEnabled = "enabled"
```

Comment:
I wonder how we workaround leader election with KO approach for KIC 🤔 but it's not the problem of this PR 😄

Reply:
I think we won't enable leader election for KIC instances as they should not interfere with each other. We will rely on KO's leader election to make sure no multiple KO instances are spawning KIC per CP. But yeah, let's tackle this when it becomes relevant. 😄